.gitignore

```diff
@@ -12,3 +12,4 @@ target/
 build-storybook.log
 logs/
 storybook-static/
+package-lock.json
```

Comment:
We probably don't need this line and it might cause a little confusion - it may seem that this is a project that uses `npm` but we don't commit our lock file, but we actually use `yarn` and _do_ commit our lockfile!